dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesRun.java

```diff
@@ -0,0 +1,390 @@
+                getMain(), new KubernetesExport.ExportConfigurer(
+                        runtime,
+                        quarkusVersion,
+                        true,
+                        true,
+                        false,
+                        workingDir,
+                        List.of(filePaths),
+                        gav,
+                        true,
+                        true,
+                        false,
+                        false,
+                        "off"));
+
+        export.image = image;
+        export.imageRegistry = imageRegistry;
+        export.imageGroup = imageGroup;
+        export.traitProfile = traitProfile;
+        export.dependencies = dependencies;
+        export.properties = properties;
+        export.configs = configs;
+        export.resources = resources;
+        export.openApis = openApis;
+        export.envVars = envVars;
+        export.volumes = volumes;
+        export.connects = connects;
+        export.annotations = annotations;
+        export.labels = labels;
+        export.traits = traits;
+
+        printer().println("Exporting application ...");
+
+        int exit = export.export();
+        if (exit != 0) {
+            printer().println("Project export failed!");
+            return exit;
+        }
+
+        if (output != null) {
+            if (RuntimeType.quarkus == runtime) {
+                exit = buildQuarkus(workingDir);
+            } else if (RuntimeType.springBoot == runtime) {
+                exit = buildSpringBoot(workingDir);
+            }
+
+            if (exit != 0) {
+                printer().println("Project build failed!");
+                return exit;
+            }
+
+            File manifest;
+            switch (output) {
+                case "yaml" -> manifest = new File(workingDir, "target/kubernetes/kubernetes.yml");
+                case "json" -> manifest = new File(workingDir, "target/kubernetes/kubernetes.json");
+                default -> {
+                    printer().printf("Unsupported output format '%s' (supported: yaml, json)%n", output);
+                    return -1;
+                }
+            }
+
+            try (FileInputStream fis = new FileInputStream(manifest)) {
+                printer().println(IOHelper.loadText(fis));
+            }
+
+            return 0;
+        }
+
+        if (RuntimeType.quarkus == runtime) {
+            exit = deployQuarkus(workingDir);
+        } else if (RuntimeType.springBoot == runtime) {
+            exit = deploySpringBoot(workingDir);
+        }
+
+        if (exit != 0) {
+            printer().println("Deployment to Kubernetes failed!");
+            return exit;
+        }
+
+        if (dev) {
+            DefaultCamelContext reloadContext = new DefaultCamelContext(false);
+            configureFileWatch(reloadContext, export, workingDir);
+            reloadContext.start();
+        }
+
+        if (dev || wait || logs) {
```

Comment:
If you run in `--dev` mode, does the pod autoatic delete when you exit via ctrl + c ?

Reply:
No, it does not. This is a good idea. Let me see if I can add this, thx